tests/plugins/justify/justify.js

```diff
@@ -252,5 +252,87 @@ bender.test(
 		bot.setHtmlWithSelection( '[<p><span contenteditable="false">foo</span></p>]' );
 		bot.execCommand( 'justifyright' );
 		assert.areSame( '<p style="text-align:right;"><span contenteditable="false">foo</span></p>', bot.getData( true ) );
+	},
+
+	'test alignment on disabled elements paragraph': function() {
```

Comment:
I add unit test for `disallowedContent`.
I also create new manual test with `disallowedContent` just to be sure :)